### DISCUSSION TITLE
Django integration

### DISCUSSION THREAD

**Discussion by ivan7707** [mirumee/ariadne#1023] [Q&A] [answered]
Hi All,

As the https://github.com/reset-button/ariadne_django package is looking for a maintainer, is there a different best practice for using ariadne with Django?

**Comment by rafalp** [comment]
I guess we can take it back, but keep it separate from `ariadne` proper? 🤔

**Comment by rafalp** [comment, marked as answer]
Just writing that `ariadne-django` is now back under Mirumee organization. @markedwards offered to be an maintainer for it as at Mirumee we currently don't have any Django projects using Ariadne.
